### PR TITLE
Motorcycle

### PR DESCRIPTION
**Pull request by Skaurm**
Added Motorcycle Class

### CODE REVIEW

File: src/test/java/MotorcycleTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public void setUp() throws Exception {
+        h = new Motorcycle();
+    }
+
+    @Test(timeout = 50)
+    public void TestGetMaxSpeed() {
+        assertEquals(100, h.getMaxSpeed());
+    }
+
+
```
Comment (shayaanmk) [inline src/test/java/MotorcycleTest.java]: Need to add test case for TestUpgradeSpeed()

**Review by shayaanmk — CHANGES_REQUESTED** [review]

File: src/test/java/MotorcycleTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * You should NOT modify this file!
+ */
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+
+public class MotorcycleTest {
+    Motorcycle h;
```
Comment (mchan7841) [inline src/test/java/MotorcycleTest.java]: Improper variable naming convention

File: src/main/java/Motorcycle.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/* A sample class that implements the given interfaces.
```
Comment (mchan7841) [inline src/main/java/Motorcycle.java:1]: No documentation throughout any of the code.

**Review by shayaanmk — APPROVED** [review]
Looks all good now

**Review by mchan7841 — APPROVED** [review]

**Pull request merged**